models/model.js

```diff
@@ -129,5 +129,4 @@ exports.selectCommentsByArticleId = (article_id) => {
             }
             return rows;
         })
-};
-
+};
```

Comment:
If the article doesn't have any comments, it's a 200 not a 404. Need to check the article exists first